include/deal.II/grid/grid_tools.h

```diff
@@ -158,8 +158,8 @@ namespace GridTools
    * function also works for cells that do not exist except that you make it
    * up by naming its vertices from the list.
    */
-  template <int dim>
-  double cell_measure (const std::vector<Point<dim> > &all_vertices,
+  template <int dim, int spacedim=dim>
```

Comment:
@drwells I see a problem here without C++11: http://cdash.kyomu.43-1.org/viewBuildError.php?buildid=608
We cannot have default template arguments on functions. As far as I can tell, there should be no problems to just omit the default value here. Could you please fix this in a follow-up PR?